src/overlays/actors/ovl_Bg_Dy_Yoseizo/z_bg_dy_yoseizo.c

```diff
@@ -6,7 +6,7 @@
 
 #include "z_bg_dy_yoseizo.h"
 
-#define FLAGS 0x02000030
+#define FLAGS (ACTOR_FLAG_2000000 | ACTOR_FLAG_20 | ACTOR_FLAG_10)
```

Comment:
OoT defines these in ascending order and I think MM should follow suit. Some actor flags are the same between the games (like targetting and enemy bgm) and others are not. But when it's documented and similar files are compared between the games, it would be beneficial to have the ordering the same for actor flags